pages/common/tmux.md

```diff
@@ -19,18 +19,41 @@
 
 `tmux attach`
 
-- Detach from the current session (inside a tmux session):
+INSIDE A TMUX SESSION:
```

Comment:
Every command in TLDR has the potential to be randomized, so each command will need `While running tmux.........`